web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/Plugin/Field/FieldWidget/ViewsBasicDefaultWidget.php

```diff
@@ -219,15 +219,27 @@ public function formElement(
       '#suffix' => '</div>',
     ];
 
+    $fieldOptionValue = ($items[$delta]->params) ? $this->viewsBasicManager->getDefaultParamValue('field_options', $items[$delta]->params) : [];
     $form['group_user_selection']['entity_and_view_mode']['field_options'] = [
       '#type' => 'checkboxes',
       '#options' => [
         'show_categories' => $this->t('Show Categories'),
         'show_tags' => $this->t('Show Tags'),
+        'show_thumbnail' => $this->t('Show Thumbnail (works only for List and Grid)'),
```

Comment:
@ion-eftodii We can remove the "(works only for List and Grid)" from the help text now that you have the conditional logic working.